SlamTests/Slam/SlamAPISpec.swift

```diff
@@ -34,6 +34,18 @@ class SlamAPISpec: QuickSpec {
         })
 
         describe("SlamAPI") {
+            
+            describe("Creates a new match") {
+                it("makes an HTTP request") {
+                    httpClient.setResponse(self.toJSON([]))
+                    
+                    slamApi.addMatch("Taka", playerTwo: "Emmanuel")
+                    
+                    expect(httpClient.wasRequestMade).to(beTrue())
+                    let expectedURL = NSURL(string: slamApi.getMatchesURL)
```

Comment:
I would add a second test for making sure you pass in the appropriate players. You have untested behavior in `SlamAPI`.